pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -1188,3 +1188,90 @@ func TestFairEvaluationForNodes(t *testing.T) {
 		}
 	}
 }
+
+func TestFitErrorNoSpammy(t *testing.T) {
+	tests := []struct {
+		name           string
+		nodeNum1       int
+		nodeNum2       int
+		addFalseFilter bool
```

Comment:
just `nodeNum2` being 0 should be enough

Reply:
Done